api/app_analytics/split_testing/tasks.py

```diff
@@ -0,0 +1,118 @@
+from collections import defaultdict
+from datetime import timedelta
+
+from app_analytics.models import FeatureEvaluationRaw
+from app_analytics.split_testing.helpers import gather_split_test_metrics
+from app_analytics.split_testing.models import ConversionEvent, SplitTest
+from django.conf import settings
+
+from environments.identities.models import Identity
+from environments.models import Environment
+from features.feature_types import MULTIVARIATE
+from features.models import Feature
+from task_processor.decorators import register_recurring_task
+
+if settings.USE_POSTGRES_FOR_ANALYTICS:
+
+    @register_recurring_task(run_every=timedelta(minutes=15))
+    def update_split_tests() -> None:
+        # Code is placed in below private function for testing.
+        return _update_split_tests()
+
+
+def _update_split_tests() -> None:
+    assert settings.USE_POSTGRES_FOR_ANALYTICS
+
+    features = Feature.objects.filter(
+        type=MULTIVARIATE,
+    ).prefetch_related("feature_states", "multivariate_options")
+
+    for feature in features:
+        environment_ids = feature.feature_states.all().values_list(
+            "environment_id", flat=True
+        )
+
+        qs_values_list = FeatureEvaluationRaw.objects.filter(
+            feature_name=feature.name,
+            environment_id__in=environment_ids,
+            identifier__isnull=False,
+        ).values_list("environment_id", "identifier")
+
+        # Eliminate duplicate identifiers
+        qs_values_list = qs_values_list.distinct()
+
+        environment_identifiers = defaultdict(list)
+        for env_identifier in qs_values_list:
+            environment_id, identifier = env_identifier
+            environment_identifiers[environment_id].append(identifier)
+
+        for environment_id, identifiers in environment_identifiers.items():
+            _save_environment_split_test(
+                feature=feature,
+                environment_id=environment_id,
+                identifiers=identifiers,
+            )
+
+
+def _save_environment_split_test(
+    feature: Feature, environment_id: int, identifiers: list[str]
+) -> None:
+    environment = Environment.objects.get(id=environment_id)
+    identities = Identity.objects.filter(
+        environment_id=environment_id,
+        identifier__in=identifiers,
+    )
```

Comment:
I think we should select_related the environment here for the calls to `get_hash_key` later on.